### DISCUSSION TITLE
Improve gamemode glow in home screen (maybe just a border?)

### DISCUSSION THREAD

**Discussion by tatitatai** [ppy/osu#26058] [Ideas] [answered]
![osu!_2023-12-21_11-12-47](https://github.com/ppy/osu/assets/32525896/69a6d5ea-c19f-499e-ad93-60947c5d189e)

Current glow is cutting edges and probably better to choose a bright pink (or even different colors for every mode, bright colors), pastel looks cheap next to the white symbols I think

**Comment by Gabixel** [comment]
I think a redesign for the toolbar is on the way. Be patient!

**Comment by Joehuu** [comment, marked as answer]
Glow and pink are gone in the new design, so closing.
